test/parallel/test-tls-cipher-list.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+var common = require('../common');
```

Comment:
Not seeing that in other tests. If that's what we want, we should make that change globally with all the tests.